plutus-tx/src/PlutusTx/Function.hs

```diff
@@ -0,0 +1,5 @@
+module PlutusTx.Function (fix) where
+
+fix :: forall a b. ((a -> b) -> a -> b) -> a -> b
+fix f = let ~x = f x in x
+{-# OPAQUE fix #-}
```

Comment:
Do I understand correctly that implementation of this function doesn't matter as it is going to be replaced with the `fix` from `PlutusCore.StdLib.Data.Function.fix` ?

Reply:
It doesn't matter for the plugin, but it matters in the sense that we should make the semantics consistent when running a Plinth program in Haskell vs. in the CEK machine.